acceptance/testdata/builder/container/cnb/buildpacks/hello_world/0.0.1/bin/build

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+echo "---> Hello World buildpack"
+
+# INPUT ARGUMENTS
+platform_dir=$2
+env_dir=${platform_dir}/env
+layers_dir=$1
+plan_path=$3
+
+# PLATFORM DIR
+echo "     platform_dir files:"
+ls -alR "${platform_dir}" | sed 's/^/       /'
+
+# ENV VARS
+echo "     env_dir: ${env_dir}"
+echo "     env vars:"
+if compgen -G "${env_dir}/*" > /dev/null; then
+  for var in ${env_dir}/*; do
+    declare "$(basename ${var})=$(<${var})"
+  done
+fi
+export | sed 's/^/       /'
+
+# LAYERS
+echo "     layers_dir: ${layers_dir}"
+
+# PLAN
+echo "     plan_path: ${plan_path}"
+echo "     plan contents:"
+cat ${plan_path} | sed 's/^/       /'
+echo
```

Comment:
It would be a bit trickier to validate that the build plan was passed correctly (see https://github.com/buildpacks/lifecycle/pull/648/files#r661649474 ) as each buildpack only sees a subset (the "buildpack plan") of the complete build plan. But maybe we could assert that the contents of the buildpack plan are a subset of the build plan.

Reply:
👍

Reply:
done https://github.com/buildpacks/lifecycle/commit/7233dad5192fb8318d0ec4805e74b19296e138e5